fe/fe-core/src/main/java/com/starrocks/connector/RemoteFileOperations.java

```diff
@@ -305,30 +303,29 @@ public FileStatus[] getFileStatus(Path... paths) {
         }
     }
 
-    public List<RemoteFileInfo> getRemotePartitions(List<Partition> partitions) {
+    public List<PartitionInfo> getRemotePartitions(List<Partition> partitions) {
         List<Path> paths = Lists.newArrayList();
         for (Partition partition : partitions) {
             Path partitionPath = new Path(partition.getFullPath());
             paths.add(partitionPath);
         }
         FileStatus[] fileStatuses = getFileStatus(paths.toArray(new Path[0]));
-        List<RemoteFileInfo> remoteFileInfos = Lists.newArrayList();
+        List<PartitionInfo> result = Lists.newArrayList();
         for (int i = 0; i < partitions.size(); i++) {
-            Partition partition = partitions.get(i);
-            FileStatus fileStatus = fileStatuses[i];
-            String locateName = fileStatus.getPath().toUri().getPath();
-            String fileName = PartitionUtil.getSuffixName(paths.get(i).toUri().getPath(), locateName);
-            RemoteFileDesc fileDesc = new RemoteFileDesc(fileName, "", fileStatus.getLen(),
-                    fileStatus.getModificationTime(), ImmutableList.of());
-            RemoteFileInfo.Builder builder = RemoteFileInfo.builder()
-                    .setFormat(partition.getInputFormat())
-                    .setFullPath(partition.getFullPath())
-                    .setFiles(Stream.of(fileDesc)
-                            .map(desc -> desc.setTextFileFormatDesc(partition.getTextFileFormatDesc()))
-                            .map(desc -> desc.setSplittable(partition.isSplittable()))
-                            .collect(Collectors.toList()));
-            remoteFileInfos.add(builder.build());
+            final Partition partition = partitions.get(i);
+            final FileStatus fileStatus = fileStatuses[i];
+            result.add(new PartitionInfo() {
+                @Override
+                public long getModifiedTime() {
+                    return fileStatus.getModificationTime();
```

Comment:
This is partition path modify time? is this time will change after modify the file in this partition?

Reply:
No. the old code is just to list modification time of that path.

Reply:
It's just for MV + Hive(updated on directory modifcation time).